src/js/services/profileService.js

```diff
@@ -433,6 +433,39 @@ angular.module('copayApp.services')
       });
     };
 
+    root.checkLatestRelease = function(cb) {
+      var releaseURL = configService.getDefaults().release.url;
+
+      root.requestLatestRelease(releaseURL, function(err, release) {
+        if (err) return cb(err);
+
+        var actualVersion = window.version;
+        var latestVersion = release.data.tag_name;
+        var majorOfActualVersion = actualVersion.split('.')[0];
+        var majorOfLatestVersion = latestVersion.split('.')[0].slice(1);
```

Comment:
This is kind of error prone. It shouldn't break if `tag_name` does not have the form `vX.X.X`.